### PR TITLE
ecf_calc plotting updated to matplotlib

### PR DESCRIPTION
**Pull request by nplee**
replace pychips with matplotlib in ecf_calc for plotting.

### CODE REVIEW

File: ciao-4.11/contrib/bin/ecf_calc [hunk 1]
```diff
@@ -365,10 +364,15 @@ def get_ecf(infile,reg,outfile,x,y,radius,binsize,fraction,clobber):
...
 def plot_ecf(rad_file):
 
-    import pychips
+    import matplotlib
+    from matplotlib import pyplot as plt
+    from matplotlib.ticker import ScalarFormatter
+
+    matplotlib.use("TkAgg")
+    matplotlib.interactive(True)
+    matplotlib.rcParams["toolbar"] = "None"
```
Comment (DougBurke) [inline ciao-4.11/contrib/bin/ecf_calc]: I'm too lazy to RTFD, but is this script intended to just create an output file, or display the data for the user to see before exiting the script.

Comment (nplee) [reply ciao-4.11/contrib/bin/ecf_calc]: It's just a plot to the screen, there's no output to file.

**Comment by DougBurke** [comment]
The ahelp file will need updating (to say Matplotlib rather than ChIPS and to add a "changes in this version ..." section).

File: ciao-4.11/contrib/bin/ecf_calc (hunk 1 above)
Comment (DougBurke) [reply ciao-4.11/contrib/bin/ecf_calc]: Is there a reason to turn the toolbar off? Users might want to be able to save a png or zoom around in the figure (or am I over-thinking it)?

**Comment by DougBurke** [comment]
Along with the changes in #150 I'd like to see the `import six` line removed - it is only used for `six.moves.input` which can be replaced with `input`.

**Comment by nplee** [comment]
@DougBurke: I've updated the ahelp, replaced six.moves.input with input, and enabled the matplotlib toolbar.

File: ciao-4.11/contrib/share/doc/xml/ecf_calc.xml [hunk 2]
```diff
@@ -191,17 +191,17 @@
...
-<!--//    <ADESC title="Changes in the December 2012 release">
+    <ADESC title="Changes in the December 2012 release">
       <PARA>
 	The script is new for in CIAO 4.5.
       </PARA>
-    </ADESC> //-->
+    </ADESC>
 
-    <ADESC title="Changes in the scripts 4.9.3 (May 2017) release">
+    <ADESC title="Changes in the June 2019 release">
```
Comment (DougBurke) [inline ciao-4.11/contrib/share/doc/xml/ecf_calc.xml]: @nplee - ta for this.

Can you move this before the Dec 2012 one (so we get the most-recent change first), and change the 2019 title to "Changes in the 4.11.4 (2019) release" [don't change the 2012 one]? I'm not sure what month this is going out in so am leaving it as `(2019)` for now.

**Comment by DougBurke** [comment]
@nplee - do you think you can put the #150 changes in this pr or would you rather do them separately?

**Comment by DougBurke** [comment]
I'm wondering if we still need the use of `input`, since we could just rely on the user closing the matplotlib window to indicate that they have finished. This would probably require removing setting the `interactive` setting for mpl.

You'd still probably need a screen message to indicate that the script was waiting. I also wonder if it would be annoying (since you have to use the mouse to close the window rather than just pressing return).

**Pull request merged**